include/libopencm3/gd32/f1x0/rcc.h

```diff
@@ -393,6 +398,16 @@
 #define RCC_CFGR3_ADCSW				(1 << 8)
 #define RCC_CFGR3_CECSW				(1 << 6)
 
+/* --- RCC_CR2 values ------------------------------------------------------ */
+
+#define RCC_CR2_HSI14CAL_SHIFT			8
+#define RCC_CR2_HSI14CAL			(0xFF << RCC_CR2_HSI14CAL_SHIFT)
+#define RCC_CR2_HSI14TRIM_SHIFT			3
+#define RCC_CR2_HSI14TRIM			(31 << RCC_CR2_HSI14TRIM_SHIFT)
```

Comment:
"31" here is unusual, it would normally be hex, and normally be _MASK and unshifted.